asv_bench/benchmarks/hdfstore_bench.py

```diff
@@ -90,6 +90,15 @@ def time_query_store_table(self):
         stop = self.df2.index[15000]
         self.store.select('table', where="index > start and index < stop")
 
+    def time_store_repr(self):
+        repr(self.store)
```

Comment:
6 is plenty of nodes to show the issue:

    before     after       ratio
  [75c8698e] [a5016b44]
-   24.82ms     7.09μs      0.00  hdfstore_bench.HDF5.time_store_repr
-   24.45ms     6.76μs      0.00  hdfstore_bench.HDF5.time_store_str